queries.yaml

```diff
@@ -160,6 +160,7 @@ post_process:
         id: true
         source: true
         boundary: "yes"
+      snap_tolerance: 0.125
```

Comment:
I should add, 0.125 Mercator "meters" is <1px until zoom 21. I certainly didn't see any visual artefacts from this when I checked down to z19. If we want to fix it for >z21, then we'll need to take another approach - snapping with a smaller distance leaves the gap intact.